fyodor/src/main/java/life/genny/fyodor/utils/FyodorUltra.java

```diff
@@ -112,6 +112,9 @@ public Page fetchWithAttributes(SearchEntity searchEntity) {
 				if (attributeCode.startsWith("_")) {
 					// handle asociated columns
 					ea = getAssociatedColumnValue(baseEntity, attributeCode);
+					// set attr codes to associated code
+					ea.setAttributeCode(attributeCode);
```

Comment:
Can these be tied down to the one method in EntityAttribute?

Reply:
They should be, but not a job for right now.